content/pt-br/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,144 @@
+
+```shell
+kubectl get pod run-as-username-pod-demo
+```
+
+Abra um shell para o contêiner em execução:
+
+```shell
+kubectl exec -it run-as-username-pod-demo -- powershell
+```
+
+Verifique se o shell está executando com o nome de usuário correto:
+
+```powershell
+echo $env:USERNAME
+```
+
+A saída deve ser:
+
+```
+ContainerUser
+```
+
+## Defina o nome de usuário para o contêiner
+
+Para especificar o nome de usuário com o qual executar os processos de um contêiner, 
+inclua o campo `SecurityContext` ([SecurityContext] 
+(/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)) 
+no manifesto do contêiner, e dentro dele, o campo `WindowsOptions` 
+([WindowsSecurityContextOptions] (/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)) 
+contendo o campo `runAsUserName`.
+
+As opções de contexto de segurança do Windows que você especificar para um contêiner, 
+se aplicam apenas a esse contêiner individual, e substituem as configurações feitas 
+no nível do Pod.
+
+Aqui está o arquivo de configuração para um pod que possui um contêiner, 
+e o campo `runAsUserName` está definido no nível do Pod e no nível do contêiner:
+
+{{< codenew file="windows/run-as-username-container.yaml" >}}
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/windows/run-as-username-container.yaml
+```
+
+Verifique se o contêiner do Pod está em execução:
+
+```shell
+kubectl get pod run-as-username-container-demo
+```
+
+Abra um shell para o contêiner em execução:
+
+```shell
+kubectl exec -it run-as-username-container-demo -- powershell
+```
+
+Verifique se o shell está executando o usuário correto, (aquele definido no nível do contêiner):
+
+```powershell
+echo $env:USERNAME
+```
+
+A saída deve ser:
+
+```
+ContainerAdministrator
+```
+
+## Limitações de nomes de usuários no Windows
+
+Para usar esse recurso, o valor definido no campo `runAsUserName` deve ser um nome 
+de usuário válido. Deve ter o seguinte formato: `DOMAIN\USER`, onde ` DOMAIN\` 
+é opcional. Os nomes de usuário do Windows não diferenciam letras maiúsculas 
+e minúsculas. Além disso, existem algumas restrições em relação ao `DOMÍNIO` e `USUÁRIO`:
+- O campo `runAsUserName`: não pode estar vazio, e não pode conter caracteres 
+  de controle (Valores ASCII : `0x00-0x1F`, `0x7F`)
+- O nome de `DOMÍNIO` NetBios, ou um nome de DNS: cada um com suas próprias restrições:
+  - Nomes NetBios: máximo de 15 caracteres, não podem iniciar com `.` (ponto), 
+  e não podem conter os seguintes caracteres: `\ / : * ? " < > |`
+  - Nomes DNS: máximo de 255 caracteres, contendo apenas caracteres alfanuméricos, 
+  pontos, e traços, e não podem iniciar ou terminar com um `.` (ponto) ou `-` (traço).
+- O `USUÁRIO`: deve ter no máximo 20 caracteres, não pode conter *somente* pontos ou espaços, 
+  e não pode conter os seguintes caracteres: `" / \ [ ] : ; | = , + * ? < > @`.
+
```

Comment:
Since you didn't translate `DOMAIN` and `USER` in the first sentence, for consistency, these should be kept non-translated for the remaining references.